src/components/Server.vue

```diff
@@ -272,6 +280,9 @@ export default {
         }
       );
     },
+    isLoading() {
+      return this.polling ? this.loadingOverride : this.Loading;
```

Comment:
`this.loading` instead of `this.Loading`

Reply:
Updated. Thanks